priv/repo/migrations/20241112150001_increase_collections_items.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Lightning.Repo.Migrations.IncreaseCollectionsItems do
+  use Ecto.Migration
+
+  def change do
+    alter table(:collections_items, primary_key: false) do
+      modify :value, :string, size: 1_000_000, from: {:string, size: 255}
+    end
```

Comment:
Please note I had to change this migration, the previous one:

```
modify :value, :string, length: 1_000_000
```

Didn't change the column, nor would it be reversible.